src/client/java/teammates/client/scripts/DataMigrationForTimezoneFieldInCourses.java

```diff
@@ -0,0 +1,136 @@
+        map("4.5", "Asia/Kabul");
+        map("5.0", "Asia/Tashkent");
+        map("5.5", "Asia/Colombo");
+        map("5.75", "Asia/Kathmandu");
+        map("6.0", "Asia/Almaty");
+        map("6.5", "Asia/Rangoon");
+        map("7.0", "Asia/Jakarta");
+        map("8.0", "Asia/Singapore");
+        map("8.75", "Australia/Eucla");
+        map("9.0", "Asia/Tokyo");
+        map("9.5", "Australia/Adelaide");
+        map("10.0", "Australia/Canberra");
+        map("10.5", "Australia/Lord_Howe");
+        map("11.0", "Pacific/Noumea");
+        map("12.0", "Pacific/Auckland");
+        map("12.75", "Pacific/Chatham");
+        map("13.0", "Pacific/Tongatapu");
+        map("14.0", "Pacific/Kiritimati");
+    }
+    
+    private static final Logic logic = new Logic();
+    
+    private boolean isPreview = true;
+    
+    public static void map(String doubleTimezone, String timezoneId) {
+        timeZoneDoubleToIdMapping.put(doubleTimezone, timezoneId);
+    }
+    
+    public static void main(String[] args) throws IOException {
+        DataMigrationForTimezoneFieldInCourses migrator = new DataMigrationForTimezoneFieldInCourses();
+        migrator.doOperationRemotely();
+    }
+    
+    @Override
+    protected void doOperation() {
+        Datastore.initialize();
+        
+        List<CourseAttributes> allCourses = getAllCoursesWithoutTimeZone();
+        for (CourseAttributes course : allCourses) {
+            updateTimezoneForCourse(course);
+        }
+    }
+    
+    private void updateTimezoneForCourse(CourseAttributes course) {
+        List<FeedbackSessionAttributes> sessions = logic.getFeedbackSessionsForCourse(course.getId());
+        String timeZone = sessions.isEmpty() ? Const.DEFAULT_TIMEZONE
+                                             : getTimeZoneId(sessions.get(0).getTimeZone());
+        
+        if (isPreview) {
+            System.out.println("Course " + course.getId() + " to be set to " + timeZone + ".");
+            return;
+        }
+        
+        try {
+            course.setTimeZone(timeZone);
+            logic.updateCourse(course);
+            System.out.println("Course + " + course.getId() + " timezone successfully set to " + timeZone + ".");
+        } catch (InvalidParametersException | EntityDoesNotExistException e) {
+            System.out.println("Failed to set timezone for course + " + course.getId() + ".");
+            e.printStackTrace();
+        }
+    }
+    
+    private String getTimeZoneId(double timeZoneDouble) {
+        return timeZoneDoubleToIdMapping.get(Double.toString(timeZoneDouble));
+    }
+    
+    private PersistenceManager getPm() {
+        return Datastore.getPersistenceManager();
+    }
+    
+    private List<CourseAttributes> getAllCoursesWithoutTimeZone() {
+        List<CourseAttributes> courseAttributes = new ArrayList<CourseAttributes>();
+        List<Course> courseEntities = getAllCourseEntities();
+        for (Course course : courseEntities) {
```

Comment:
If the above suggestion is taken, `course` could be renamed as `courseEntity` to avoid a name clash with `courses` which represents a list of `CourseAttributes`